FirebaseRemoteConfig/Sources/RCNConfigSettings.m

```diff
@@ -110,6 +110,7 @@ - (instancetype)initWithDatabaseManager:(RCNConfigDBManager *)manager
     }
 
     _isFetchInProgress = NO;
+    _lastTemplateVersion = @"1";
```

Comment:
can we store this in `_userDefaultsManager`? (see `lastFetchTimeInterval` for example). i think this will allow us to avoid any DB interactions (we longer have to add templateVersion to device context)

Reply:
another thought is this shouldn't be initialized as 1 because if the developer publishes their first version then the client won't fetch?

Reply:
I don't think so, it doesn't look like it's saved to disk (https://github.com/firebase/firebase-ios-sdk/blob/96450bd48f3ca6ec7e7854a45f95d2a1449e1296/FirebaseRemoteConfig/Sources/RCNConfigSettings.m#L101). 
And that's a good point, I'll initialize it to 0.

Reply:
hmm I think NSUserDefaults does save to disk? https://developer.apple.com/documentation/foundation/nsuserdefaults I would be surprised if we reset something like `lastFetchTime` every app restart

Reply:
Oh I thought you meant the manager, I updated to use userDefaults